### PR TITLE
Adjust docs for Loki removal

### PR DESCRIPTION
**Pull request by shorim**
**Description**

Changes proposed in this pull request:

- Adjust docs in for Loki removal

**Related issue(s)**

kyma-project/kyma#17637

### CODE REVIEW

**Review by chrkl — DISMISSED** [review]

File: docs/user/02-logs.md [hunk 1]
```diff
@@ -20,9 +20,8 @@ The Telemetry module provides [Fluent Bit](https://fluentbit.io/) as a log agent
...
 4. The Telemetry module configures Fluent Bit with your custom output configuration.
-5. If Kyma's [deprecated](https://kyma-project.io/blog/2022/11/2/loki-deprecation/) logging component is installed, the Telemetry Manager configures the shipment to the in-cluster Loki instance automatically.
-6. As specified in your LogPipeline configuration, Fluent Bit sends the log data to observability systems outside or inside the Kyma cluster. Here, you can use the integration with HTTP to integrate a system directly or with an additional Fluentd installation.
-7. The user accesses the internal and external observability system to analyze and visualize the logs.
+5. As specified in your LogPipeline configuration, Fluent Bit sends the log data to observability systems outside or inside the Kyma cluster. Here, you can use the integration with HTTP to integrate a system directly or with an additional Fluentd installation.
+6. The user accesses the internal and external observability system to analyze and visualize the logs.
```
Comment (NHingerl) [inline docs/user/02-logs.md:24]: Is "the user" someone else than the "you" we address in the previous step?

Comment (NHingerl) [reply docs/user/02-logs.md:24]:
```suggestion
6. To analyze and visualize your logs, access the internal and external observability system.
```

Comment (NHingerl) [reply docs/user/02-logs.md:24]: wait, internal AND external? Or rather "or" ?

Comment (shorim) [reply docs/user/02-logs.md]: The user can have only an internal observability system or only an external observability system or both. But I think "or" here fits better so I changed it.

**Review by NHingerl — APPROVED** [review]

**Pull request merged**